feature-libs/customer-ticketing/root/feature-name.ts

```diff
@@ -0,0 +1 @@
+export const CUSTOMER_TICKETING_FEATURE = 'customer-ticketing';
```

Comment:
create a constant for CUSTOMER_TICKETING_CORE_FEATURE

it is going to be used to bundle the components with core when we ship this feature lib. It is also used during the proxy facade's creation